examples/produce_part_brep/README.md

```
@@ -0,0 +1,22 @@
+PartCAD example project to demonstrate parts defined using BREP files.
+
+## Usage
+```shell
+pc inspect box
+```
+
+
+## Parts
+
+### box
+<table><tr>
```

Markdownlint (0.37.0)

Element: table
Inline HTML

(MD033, no-inline-html)